README.md

```diff
@@ -34,20 +36,25 @@ Ensure your PowerVS instance is prepped for deploying OpenShift Clusters. Please
 * Sets up the latest Terraform binary
 * Provides interactive mode to populate required Terraform variables
 * Abstract out the Terraform lifecycle management
-
-
+&nbsp;
+   [![asciicast](https://asciinema.org/a/380928.svg)](https://asciinema.org/a/380928)
```

Comment:
Can we move this embedded frame to other section? may be at the bottom or a new md file. It occupies too much space of the README.

Reply:
Done